draft-ietf-httpbis-resumable-upload.md

```diff
@@ -724,9 +724,13 @@ Content digests help verify the integrity of the content in an individual reques
 
-# Subsequent Resources
+# Responses to Uploads
 
-The server might process the uploaded representation data and make its results available in another resource during or after the upload. This subsequent resource is different from the upload resource created by the upload creation request ({{upload-creation}}). The subsequent resource does not handle the upload process itself, but instead facilitates further interaction with the uploaded representation data. The server MAY indicate the location of this subsequent resource by including the `Content-Location` header field in the interim or final responses generated while creating ({{upload-creation}}), appending to ({{upload-appending}}), or retrieving the offset ({{offset-retrieving}}) of an upload. For example, a subsequent resource could allow the client to fetch information extracted from the uploaded representation data.
+Conventional HTTP uploads often include information about the uploaded representation in the corresponding response header and/or content. The same is possible for resumable uploads, in that the server can include such information in the response to the request that transferred the remaining representation data and included the `Upload-Complete: ?1` header field. Clients can regard this response as the final response for the entire upload, as detailed in {{upload-creation}} and {{upload-appending}}.
+
+However, due to network interruptions, the upload resource might receive the remaining representation data, complete the upload, and send a response to the client, which then might not receive the response due to the interruption. The client can learn that the upload is complete by retrieving its state ({{offset-retrieving}}), but resumable uploads as defined in this document do not offer the ability to recover the missed response.
+
+To address this issue, the server can choose to send the corresponding information in header fields, which are then included in the final response, once all representation data has been transferred, as well as responses when the client fetches the upload state via a HEAD request ({{offset-retrieving}}). This way, the client can retrieve the information from the header even if it failed to receive the final response. In addition, if the server responds with a representation and wants to make it available for clients for later retrieval as well, it can include the `Content-Location` header field in the final response and responses when fetching the upload state. The client can then retrieve the representation from the provided URI, even if it failed to receive the final response.
```

Comment:
I found this section a bit hard to read. Could we simplify it? 

Suggestion:
> To fix this issue, the server can send the needed information in headers that are included in both the final response 
 and also in responses when the client queries the upload state with a HEAD request. Additionally, if the server sends a representation and wants to make it available for later, it can include the `Content-Location` header field in both the final response and in responses when checking the upload state. This allows the client to get the information even if it didn't receive the final response.

Reply:
Thanks, I incorporated this into the PR